src/store/singing.ts

```diff
@@ -1874,144 +1871,171 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
           }
-        }
-        if (limiter) {
-          channelStrip.output.connect(limiter.input);
-          limiter.output.connect(clipper.input);
-        } else {
-          channelStrip.output.connect(clipper.input);
-        }
-        clipper.output.connect(offlineAudioContext.destination);
-
-        // スケジューリングを行い、オフラインレンダリングを実行
-        // TODO: オフラインレンダリング後にメモリーがきちんと開放されるか確認する
-        offlineTransport.schedule(renderStartTime, renderDuration);
-        const audioBuffer = await offlineAudioContext.startRendering();
-        const waveFileData = convertToWavFileData(audioBuffer);
-
-        try {
-          await window.electron
-            .writeFile({
-              filePath,
-              buffer: waveFileData,
-            })
-            .then(getValueOrThrow);
-        } catch (e) {
-          window.electron.logError(e);
-          if (e instanceof ResultError) {
+          clipper.output.connect(offlineAudioContext.destination);
+
+          // スケジューリングを行い、オフラインレンダリングを実行
+          // TODO: オフラインレンダリング後にメモリーがきちんと開放されるか確認する
+          offlineTransport.schedule(0, renderDuration);
+          const audioBuffer = await offlineAudioContext.startRendering();
+          const waveFileData = convertToWavFileData(audioBuffer);
+
+          try {
+            await window.electron
+              .writeFile({
+                filePath,
+                buffer: waveFileData,
+              })
+              .then(getValueOrThrow);
+          } catch (e) {
+            window.electron.logError(e);
+            if (e instanceof ResultError) {
+              return {
+                result: "WRITE_ERROR",
+                path: filePath,
+                errorMessage: generateWriteErrorMessage(e),
+              };
+            }
             return {
-              result: "WRITE_ERROR",
+              result: "UNKNOWN_ERROR",
               path: filePath,
-              errorMessage: generateWriteErrorMessage(e),
+              errorMessage:
+                (e instanceof Error ? e.message : String(e)) ||
+                "不明なエラーが発生しました。",
             };
           }
-          return {
-            result: "UNKNOWN_ERROR",
-            path: filePath,
-            errorMessage:
-              (e instanceof Error ? e.message : String(e)) ||
-              "不明なエラーが発生しました。",
-          };
-        }
 
-        return { result: "SUCCESS", path: filePath };
+          return { result: "SUCCESS", path: filePath };
+        };
+
+        commit("SET_EXPORTING_AUDIO", { exportingAudio: true });
+        return exportWaveFile().finally(() => {
+          commit("SET_CANCELLATION_OF_AUDIO_EXPORT_REQUESTED", {
+            cancellationOfAudioExportRequested: false,
+          });
+          commit("SET_EXPORTING_AUDIO", { exportingAudio: false });
+        });
       }
     ),
   },
+
+  CANCEL_AUDIO_EXPORT: {
+    async action({ state, commit }) {
+      if (!state.exportingAudio) {
+        return;
+      }
```

Comment:
ここエラーの握り潰しになっているかもです。
といっても偶然入れ違いで呼ばれることもあるとは思うので、とりあえずwarnを書いておくのはどうでしょう？
```suggestion
    async action({ state, commit, dispatch }) {
      if (!state.exportingAudio) {
        dispatch("LOG_WARN", "CANCEL_AUDIO_EXPORT on !exportingAudio");
        return;
      }
```